src/python/grapl-common/grapl_common/env_helpers.py

```diff
@@ -130,6 +160,8 @@ def _localstack_client(
     return client_create_fn(
         service,
         endpoint_url=f"http://{os.environ['LOCALSTACK_HOSTNAME']}:{os.environ['EDGE_PORT']}",
+        region_name=region,
+        config=config,
```

Comment:
^ was this resolved?

Reply:
No, we can probably remove region here? The issue I ran into is effectively the fact that graplctl's idea of a "grapl region" isn't the same thing as AWS_REGION and/or AWS_DEFAULT_REGION. So I'm not sure what the "right" way to reconcile all this is, but we can certainly remove this `region` param and fix the consequences. Injecting the `config` here is important because we use it in various contexts to tweak timeouts.